random_numbers/Makefile.am

```diff
@@ -23,19 +23,18 @@
 # Ed Hartnett 2/28/19
 
 # Include .h and .mod files.
-AM_CPPFLAGS = -I${top_srcdir}/include
-AM_CPPFLAGS += -I${top_builddir}/time_manager
+AM_CPPFLAGS = -I${top_builddir}/.mod -I${top_srcdir}/include
 
 # Build these uninstalled convenience library.
-noinst_LTLIBRARIES = libMersenneTwister.la librandom_numbers.la
+noinst_LTLIBRARIES = libmersennetwister.la librandom_numbers.la
 
 # Each convenience library depends on its source.
 librandom_numbers_la_SOURCES = random_numbers.F90
-libMersenneTwister_la_SOURCES = MersenneTwister.F90
+libmersennetwister_la_SOURCES = mersennetwister.F90
```

Comment:
Why change the name of this file?

Reply:
I couldn't get the .mod file to copy over to the common directory if the name of the module was different from the name of the source file